vesc_driver/src/vesc_driver.cpp

```diff
@@ -190,7 +246,7 @@ void VescDriver::vescErrorCallback(const std::string& error)
  */
 void VescDriver::dutyCycleCallback(const std_msgs::Float64::ConstPtr& duty_cycle)
 {
-  if (driver_mode_ == MODE_OPERATING)
+  if (driver_mode_ = MODE_OPERATING)
```

Comment:
This has to be incorrect. It should definitely be a comparison operator (`==`) and not an assignment operator (`=`).